### ISSUE TITLE
Server Whitelist regular expressions

### ISSUE THREAD

**Issue by jastax** [karakun/OpenWebStart#260]
Just a short question:

Would it be possible to extend the server whitelist with regular expressions?

Thanks

**Comment by sclassen** [comment]
Could you give a short example of what you want to enter?
Maybe some simple wildcard character is sufficient and most likely easier to apply than a full regexp

**Comment by jastax** [comment]
Hi @sclassen,
thanks for your answer.
Using wildcard characters would be sufficient for our usecase.

**Comment by sclassen** [comment]
Great to hear. Could you give one or two examples, similar to:
```
https://mydomain.com:*
https://*.mydomain.com
http://192.168.*.*
```

**Comment by jastax** [comment]
https://*.mydomain.com

**Comment by janakmulani** [comment]
we are already implementing this. Please see https://github.com/AdoptOpenJDK/IcedTea-Web/pull/674

**Comment by Jamila891** [comment]
has been implemented and is available in version 1.2.1. See documentation: https://github.com/karakun/OpenWebStart/blob/master/documentation/guide/OWSGuide.adoc#server-whitelist
